src/main/java/net/dv8tion/jda/core/JDA.java

```diff
@@ -743,34 +735,42 @@ public boolean equals(Object o)
 
     /**
      * Shuts down JDA, closing all its connections.
-     *
-     * <p>This is the same as calling {@link #shutdown(boolean) shutdown(true)}.
+     * After this command is issued the JDA Instance can not be used anymore.
+     * Already enqueued {@link net.dv8tion.jda.core.requests.RestAction RestActions} are still be executed.
      */
     void shutdown();
```

Comment:
Are they actually still executed?
Requester#request rejects new requests after the rateLimiter is shutdown

Also `are still be executed` makes no sense as a statement.